client/index.html

```diff
@@ -4,7 +4,6 @@
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>%VITE_APP_TITLE%</title>
```

Comment:
Czy jeśli usunąłeś tą zmienną z .env.development to powinna tu być?

Reply:
kochany, ta linijka została usunięta